modules/apps/portal-search-elasticsearch6/portal-search-elasticsearch6-impl/src/main/java/com/liferay/portal/search/elasticsearch6/internal/sort/DefaultSortTranslator.java

```diff
@@ -69,7 +69,12 @@ public void translate(
 
 			SortOrder sortOrder = SortOrder.ASC;
 
-			if (sort.isReverse() || sortFieldName.equals("_score")) {
+			String originalSortName = sort.getFieldName();
+
+			if (sort.isReverse() ||
+				(sortFieldName.equals("_score") &&
+				 !sortFieldName.equals(originalSortName))) {
```

Comment:
Remove this check - see comment at https://github.com/liferay-search/liferay-portal/pull/10/files#diff-0fd9f35f159ee8c153ccbb56758aeb2cR76